img/foxsi_image_det.pro

```diff
@@ -14,6 +14,7 @@
 ;				Default entire flight
 ;		THR_N	Threshold to use for n-side data.
 ;		YEAR	2012 or 2014 flight.  Default 2014.
+;		CDTE Set to 1 if CdTe detector
```

Comment:
Is this valid for both, FOXSI2 and FOXSI3? It seems that by setting this keyword you don't need to worry about flipping the image. Is this true for CdTe FOXSI2 and FOXSI3 detectors? If not, I think we would need to consider that aspect here in the code.

Reply:
Yes, it is true for both FOXSI-2 and FOXSI-3. Those detectors do not have the same orientation, but this difference it taken care off when creating the level 1 data. So, when calling this procedure, which use the level 2 data, we are at a point where it works the same for FOXSI-2 and FOXSI-3 CdTe